p5js/PoseNet/sketch.js

```diff
@@ -19,12 +19,18 @@ function setup() {
   video.size(width, height);
 
   // Create a new poseNet method with a single detection
-  poseNet = ml5.poseNet(video, 'single', gotPoses);
-
+  poseNet = ml5.poseNet(video, modelReady);
+  poseNet.on('pose', function (results) {
```

Comment:
This is a bit different than the rest of ml5 style. Are there many other events besides `'post'`? I might suggest something like?

```javascript
poseNet.detect(gotPoses);

function gotPoses(results) {

}
```

Reply:
Yes, this one has an Event Emitter and it's a bit different from the other methods. But I do like the idea of having events being emmited instead of having to create a manual loop with `gotPoses()`. Since most of the time you will want to use this method in a continuos way.

Reply:
Yes, that makes sense. I guess my question is -- are there more events than just `'pose'`? If it's just the one I might skip the step of passing in a string?

Reply:
That's the only one for now. But you need to pass it the event name you want to listen to:
https://nodejs.org/api/events.html#events_emitter_on_eventname_listener

Reply:
Ok! All makes sense, happy to keep as is to match the event emitter style.